src/ray/raylet/scheduling/scheduling_ids.h

```diff
@@ -88,6 +88,8 @@ enum class SchedulingIDTag { Node, Resource };
 template <SchedulingIDTag T>
 class BaseSchedulingID {
  public:
+  explicit BaseSchedulingID() = default;
```

Comment:
hmm this is error-prone. should we initialize as -1?

Reply:
The `id_` is already initialized in `BaseSchedulingID`.
`int64_t id_ = -1;`